src/pages/product-list/components/display/product-item/product-item.tsx

```diff
@@ -91,45 +155,106 @@ const ProductItem = (props: IProductItem) => {
                 <Typography>
                   <Hidden mdDown>
                     <Box fontFamily="RobotoBold" fontSize="18px">
-                      {props.title.length > 50 ? props.title.substring(0, 50) + '...' : props.title}
+                      {props.title.length > 100 ? props.title.substring(0, 100) + '...' : props.title}
                     </Box>
                   </Hidden>
                   <Hidden mdUp>
                     <Box fontFamily="RobotoRegular" fontSize="14px">
-                      {props.title.length > 50 ? props.title.substring(0, 50) + '...' : props.title}
+                      {props.title.length > 100 ? props.title.substring(0, 100) + '...' : props.title}
                     </Box>
                   </Hidden>
                 </Typography>
               </Grid>
               <Grid container xs={12}>
-                <Grid item xs={6}>
-                  <Typography variant="h4">
-                    <Hidden mdDown>
-                      <Box fontFamily="OswaldMedium" fontSize="24px" mt={3}>
-                        {props.price}
-                      </Box>
-                    </Hidden>
-                    <Hidden mdUp>
-                      <Box fontFamily="OswaldMedium" fontSize="18px" mt={3}>
-                        {props.price}
-                      </Box>
-                    </Hidden>
-                  </Typography>
-                </Grid>
-                <Grid item xs={6}>
-                  <Hidden mdDown>
-                    <Box>
-                      {/* TODO this needs to be a link*/}
-                      <StyledLogo src={props.logo} />
-                    </Box>
-                  </Hidden>
-                  <Hidden mdUp>
-                    <Box mt={3}>
-                      {/* TODO this needs to be a link*/}
-                      <StyledLogo src={props.logo} />
-                    </Box>
-                  </Hidden>
-                </Grid>
+                {props.discountPrice ? (
```

Comment:
why same thing twice?

Reply:
One is for the list where we have image, and the other one is for the list where we do not have image

Reply:
yeah but why the logic is not deep down where the image is rendered? right now the whole content is being duplicated for one small thing as I can see

Reply:
We'll have to discuss this on skype